packages/programs/mud.config.ts

```diff
@@ -0,0 +1,54 @@
+import { defineWorld } from "@latticexyz/world";
+
+export default defineWorld({
+  // Note: this is required as the world is deployed with this
+  deploy: {
+    upgradeableWorldImplementation: true,
+  },
+  namespace: "default-1",
+  userTypes: {
+    ObjectTypeId: {
+      filePath: "@dust/world/src/ObjectTypeId.sol",
+      type: "uint16",
+    },
+    EntityId: { filePath: "@dust/world/src/EntityId.sol", type: "bytes32" },
+    ResourceId: {
+      filePath: "@latticexyz/store/src/ResourceId.sol",
+      type: "bytes32",
+    },
+    ProgramId: {
+      filePath: "@dust/world/src/ProgramId.sol",
+      type: "bytes32",
+    },
+  },
+  tables: {
+    Admin: {
```

Comment:
wdyt of calling it `Owner` seems more standard in the smart contract world

Reply:
while admin feels like something more global, like for the shop program there will be an admin that can set the global fee and fee recipient

Reply:
renamed!